perllib/FixMyStreet/Cobrand/Merton.pm

```diff
@@ -78,6 +78,8 @@ sub open311_extra_data_include {
 sub report_new_munge_before_insert {
     my ($self, $report) = @_;
 
+    return unless $self->{c}->stash->{report}->to_body_named('Merton');
```

Comment:
I realise this is the same as Bucks, but I don't understand why Bucks isn't using `$report` that has been passed in!

Reply:
Oh yes, that would be a bit simpler wouldn't it!